aws/cloudformation/components/alarms.yml.erb

```diff
@@ -1,25 +1,20 @@
-  DaemonStorageUtilizationAlarm:
-    Type: "AWS::CloudWatch::Alarm"
+  Critical: {Type: AWS::SNS::Topic}
+  Warning: {Type: AWS::SNS::Topic}
+<%{Warning: 70, Critical: 90}.each do |name, threshold| -%>
+  DiskUsed<%=name%>:
+    Type: AWS::CloudWatch::Alarm
     Properties:
-      AlarmActions:
-        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CDO-<%= rack_env?(:production) ? 'Urgent' : 'LowPriority'%>"
-      AlarmDescription: Send page if daemon storage utilization exceeds 80% for an hour
-      AlarmName: <%="#{stack_name}_daemon_high_storage_utilization" %>
-      ComparisonOperator: GreaterThanThreshold
-      Dimensions:
-        - Name: Filesystem
-          Value: '/dev/xvda1'
-        - Name: MountPath
-          Value: '/'
-        - Name: InstanceId
-          Value: <%= daemon_instance_id || "!Ref #{daemon}" %>
-      EvaluationPeriods: 60
-      MetricName: DiskSpaceUtilization
-      Namespace: 'System/Linux'
```

Comment:
Note: This previous alarm was still configured for the [legacy CloudWatch Monitoring Scripts](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/mon-scripts.html#monitoring-scripts-intro) (`System/Linux` namespace), which has been deprecated in favor of the [CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/metrics-collected-by-CloudWatch-agent.html#windows-metrics-enabled-by-CloudWatch-agent) (`CWAgent` namespace).

Our long-lived instances still publish the old metrics because we haven't yet explicitly uninstalled the old script, but we no longer install the old script on new instances.